package.json

```diff
@@ -85,10 +84,12 @@
     "chroma-js": "^1.2.1",
     "classnames": "^2.2.5",
     "d3": "^4.2.1",
+    "gulp-uglify-es": "^1.0.4",
     "isomorphic-fetch": "^2.2.1",
     "lodash": "^4.17.11",
     "mapbox-gl": "^0.50.0",
     "moment": "^2.14.1",
+    "openaq-data-format": "github:openaq/openaq-data-format#e3bda716006e1f69c82b8b1e98e5614688f4718e",
```

Comment:
This may just be like this for testing, but should we pin this to a commit or to something like `master`?

Reply:
Updated this to have it track master now that the pr on openaq-data-format is merged in.